migrator/clone/rocksdb/db_clone_manager_impl.go

```diff
@@ -327,27 +328,46 @@ func (d *dbCloneManagerImpl) GetDirName(cloneName string) string {
 func (d *dbCloneManagerImpl) DecommissionRocksDB() {
 	log.Info("DecommissionRocksDB")
 	for k := range d.cloneMap {
-		path := d.getPath(k)
-		if exists, err := fileutils.Exists(path); err != nil {
+		dirPath := d.getPath(k)
+		if exists, err := fileutils.Exists(dirPath); err != nil {
 			log.Error(err)
 			continue
 		} else if exists {
-			log.Infof("Removing database %s", path)
-			linkTo, err := fileutils.ResolveIfSymlink(path)
-			if err != nil {
-				log.Error(err)
-				continue
-			}
-			log.Infof("Remove path = %q", path)
-			if err = os.RemoveAll(path); err != nil {
-				log.Error(err)
-				continue
-			}
-			// Remove any rocks database if in postgres mode
-			log.Infof("Remove linkTo = %q", linkTo)
-			if err = os.RemoveAll(linkTo); err != nil {
-				log.Error(err)
-				continue
+			// TODO(ROX-9882): While there is still stuff on the PVC there is an expectation that current
+			// exists in a certain state.  So until we can remove the rest of the components on the PVC, it
+			// is safer to clear the contents.  This will keep the link and directory in place to ensure
+			// things behave in a safe manner on restarts.
+			if k == CurrentClone {
+				dir, err := os.ReadDir(dirPath)
+				if err != nil {
+					log.Error(err)
+					continue
+				}
+				for _, d := range dir {
+					log.Infof("Removing %q", d.Name())
```

Comment:
If we remove all the content here, the old version would fail scan() while reading versions.
It would be better if we can carry some information to old versions telling people/support that what happens and how to address this condition.
For example, we can remove all copies like you did before and then add a new copy of link and target dir as 
current -> .central.internal.database.decommissioned.roll.back.not.supported.... and we can communicate this signature to support.
That could make us happier.

Reply:
Failing scan wouldn't be an issue in Postgres mode as we don't fail migrator for a failed Rocks scan.  I think the safest way based on what we've been seeing is to keep whatever lingers in current except for migration_version.yaml. 
 We could remove/rename that to ensure we never consider this to be a valid DB to migrate from and change what current points to as you suggest.